src/js/components/Diagram/stories/Progressing.js

```diff
@@ -72,7 +72,7 @@ export const Progressing = () => {
               ))}
             </Box>
           </Box>
-          <Diagram connections={connections} />
+          <Diagram animation="draw" connections={connections} />
```

Comment:
I think this should transit slower.

Reply:
See if you can leverage `global.duration` to make the behavior transit slower when using `draw`

Reply:
With the newest implementation, I also allowed the user to specify a duration via an object for the animation prop:

`animation={{ type: 'draw', duration: 2000 }}`

And after testing this, it slowed down the animation to what we expected in our previous meeting. Next, I planned on trying out the `global.duration` stuff in the Progressing story that I've been using to test so far!